docs/CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ Version 3.42.0-eisop4 (April ?, 2024)
 
 **Implementation details:**
 
+New method `GenericAnnotatedTypeFactory#addComputedTypeAnnotationsWithoutFlow(Tree, AnnotatedTypeMirror)` that sets
+`useFlow` to `false` before calling `addComputedTypeAnnotations`. Subclasses should override method
+`GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)` instead.
+Deprecated the `GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror, boolean)` overload.
+Instead, set `GenericAnnotatedTypeFactory#useFlow` appropriately and call
+`GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)`.
```

Comment:
```suggestion
```

Reply:
I don't think this adds anything. If you want it, it needs some connection to `addComputedTypeAnnotationsWithoutFlow`.